baking/Cornmeal_Molasses_Scones.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "Cornmeal Molasses Scones"
+source_url: "https://www.seductioninthekitchen.com/cornmeal-molasses-scones/"
+oven_temp: "370 F"
+time: "60m"
+yield: "9 inch pie pan"
+---
+
+# Ingredients
+
+
+- 1 1/4 cup boiling water
+- 1/2 cup cornmeal
+- 1/4 cup molasses
+- 3 tablespoons butter
+- 1 teaspoon salt
+- 1 teaspoon cinnamon
+- 1 egg
+- 1 ounce package active dry yeast 1/4
+- 3 cups all-purpose flour
+
+
+
+# Directions
+
+1. Combine the water, molasses, and butter. Stir it till it looks like the molasses is dissolved. 
+
+2. Add the yeast, and let the yeast bloom for about 5 minutes. Then add the egg, and stir everything one more time.
+
+3. Next add all the dry ingredients. Combine it until you have a dough ball, cover and let it rise for about 30 minutes.
+
+4. Grease a 9 inch pie plate and spread the dough into the pie plate.
+
+5. Preheat the oven to 375 (letting the dough rise for a secondish time while preheating) and bake the scones for about 25 minutes. You know it is done when sounds hollow when you tap them.
```

Comment:
Thoughts on moving the preheat step to before step 4? Or is the "secondish time" important?

Reply:
The second rise time is important to the process, or you end up with a flat puck of sadness and not fluffy scones.

Reply:
Sounds good.